src/AccessControl/tainted.py

```diff
@@ -28,6 +28,24 @@
     from cgi import escape
 
 
+def should_be_tainted(value):
+    if isinstance(value, int):
+        return 60 == value
```

Comment:
The 60 comes out of nowhere for me. Could you add a small comment here? Just something like "ord('<') is 60".

Reply:
Yup